app/upgrades/v11/constants.go

```diff
@@ -17,10 +17,15 @@
 package v11
 
 const (
+	// UpgradeNameRC3 is the shared upgrade plan name for the RC3 testnet upgrade
+	UpgradeNameRC3 = "v11.0.0-rc3"
+	// UpgradeInfo defines the binaries that will be used for the upgrade
+	UpgradeInfoRC3 = `'{"binaries":{"darwin/arm64":"https://github.com/evmos/evmos/releases/download/v11.0.0-rc3/evmos_11.0.0-rc3_Darwin_arm64.tar.gz","darwin/amd64":"https://github.com/evmos/evmos/releases/download/v11.0.0-rc3/evmos_11.0.0-rc3_Darwin_amd64.tar.gz","linux/arm64":"https://github.com/evmos/evmos/releases/download/v11.0.0-rc3/evmos_11.0.0-rc3_Linux_arm64.tar.gz","linux/amd64":"https://github.com/evmos/evmos/releases/download/v11.0.0-rc3/evmos_11.0.0-rc3_Linux_amd64.tar.gz","windows/x86_64":"https://github.com/evmos/evmos/releases/download/v11.0.0-rc3/evmos_11.0.0-rc3_Windows_x86_64.zip"}}'`
```

Comment:
Q: For what purpose are we keeping this upgrade info in the constants.go? Since SDK v0.46 we can't just use this anymore for the upgrade proposals because the checksums have to be provided as well.

Reply:
true, we don't need it really